reana_workflow_controller/workflow_run_manager.py

```diff
@@ -358,12 +365,42 @@ def _create_job_spec(self, name, command=None, image=None,
         if os.getenv('FLASK_ENV') == 'development':
             job_controller_env_vars.extend(
                 current_app.config['DEBUG_ENV_VARS'])
+        job_controller_env_vars.extend([{
+            'name': 'REANA_USER_ID',
+            'value': owner_id
+        }])
+
+        user = User.query.filter_by(id_=owner_id).one_or_none()
+        secrets_store = REANAUserSecretsStore(user)
+        user_secrets = secrets_store.get_secrets()
+        job_controller_env_secrets = []
+
+        for secret in user_secrets:
+            name = secret['name']
+            if secret['type'] == 'env':
+                job_controller_env_secrets.append(
+                    {
+                        'name': name,
+                        'valueFrom': {
+                            'secretKeyRef': {
+                                'name': owner_id,
+                                'key': name
+                            }
+                        }
+                    })
+
         job_controller_container.env.extend(job_controller_env_vars)
+        job_controller_container.env.extend(job_controller_env_secrets)
         job_controller_container.volume_mounts = [
             {
                 'name': 'default-shared-volume',
                 'mountPath': SHARED_FS_MAPPING['MOUNT_DEST_PATH'],
             },
+            {
+                'name': owner_id,
+                'mountPath': "/etc/reana/secrets",
```

Comment:
We should extract this to configuration, also in https://github.com/reanahub/reana-job-controller/pull/158/files#diff-669c9f342cac1a455e5c652c40abf085R130.

Package configuration for now, once we merge we create an issue to move it to REANA-Commons.